### ISSUE TITLE
Fails to render when the component uses optional chaining operator

### ISSUE THREAD

**Issue by nachodd** [tkskto/vue-component-analyzer#174]
When a components uses the [optional chaining operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining), the console throws: 

> Something went wrong with reading File.vue
> Unexpected token .

**Comment by nachodd** [comment]
Similar behavior happens when in a Vue component, on the <script> tag there is a class defined, like: 
```vue
export default {
  // Vue component code...
}

class something {
  editable = false
  constructor() {
    this.editable = data.column
  }
}
``` 

error thrown: 
> Unexpected token =

**Comment by nachodd** [comment]
Nice work! 👏🏼

**Comment by tkskto** [comment]
@nachodd Thank you for your issue 🙌

I published v0.5.1, and this issue should be solved.
